pkg/obfuscate/sql_test.go

```diff
@@ -892,15 +906,15 @@ func TestSQLQuantizer(t *testing.T) {
 		},
 		{
 			"SELECT * FROM public.table ( array [ ROW ( array [ 'magic', 'foo',",
-			"SELECT * FROM public.table ( array [ ROW ( array [ ?",
+			"SELECT * FROM public.table ( array [ROW ( array [?",
 		},
 		{
 			"SELECT pg_try_advisory_lock (123) AS t46eef3f025cc27feb31ca5a2d668a09a",
 			"SELECT pg_try_advisory_lock ( ? )",
 		},
 		{
 			"INSERT INTO `qual-aa`.issues (alert0 , alert1) VALUES (NULL, NULL)",
-			"INSERT INTO qual-aa . issues ( alert0, alert1 ) VALUES ( ? )",
+			"INSERT INTO `qual-aa`.issues ( alert0, alert1 ) VALUES ( ? )",
```

Comment:
I'm totally for reducing the unnecessary spaces around `.` and `[` et al, but why are we adding quotes where there were previously none?

Reply:
This PR doesn't *add* quotes, but it preserves any quotes that were in the original query. 

The input to this test has the quotes, and the output used to not contain them. Now the output preserves the quotes.

The quotes are not always important, but sometimes they are for the query to be correct. Instead of figuring out whether the quotes are important or not, we should probably just preserve the original text.

Reply:
It _adds_ quotes because they weren't previously there in other versions, regardless of whether the original query has them or not. That's what I meant.

Was this ever an issue brought up by anyone or is it just something you personally think would be better to have?